src/cards/top-languages-card.js

```diff
@@ -245,8 +246,23 @@ const useLanguages = (topLangs, hide, langs_count) => {
     .sort((a, b) => b.size - a.size)
     .filter((lang) => {
       return !langsToHide[lowercaseTrim(lang.name)];
-    })
-    .slice(0, langsCount);
+    });
+
+  if (merge_others) {
+    // Return 'langs_count' -1 top languages and merge the rest of the languages into "others" category.
+    const others = langs.splice(langsCountClamped - 1);
```

Comment:
@RedHeadphone I just noticed a bug in my code. This only works when the user has more languages than the `langCountClamped` variable. Can you maybe make sure that it also handles the case where the user has less languages than the `langCountClmaped variable? Meaning it should simply return `langs = langs.slice(0, langsCountClamped)` if this is the case.

Reply:
This should work right?
`if (merge_others && langs.length > langsCountClamped) {`

Reply:
@RedHeadphone Yea, that syntax looks correct. If four languages are given by the API and the user-specified five languages, it would then return `langs = langs.slice(0, langsCountClamped);` which returns 4 languages. If the API gives eight languages and the user specifies 5, it would return the value with the `others` item.

Reply:
yeah added test accordingly, please review it !